### ISSUE TITLE
What version of IMGT/HLA database is being used?

### ISSUE THREAD

**Issue by seqlee** [AlexanderDilthey/MHC-PRG#23]
Hi,

I am testing HLA*PRG for benchmarking and I noticed your hla_nom_g file is from 3.24.0 release however, your rearragned files containing allele sequences (mostly contained in tmp2/GS_nextGen/hla ?) are missing some alleles.  My guess is that HLA*PRG actually uses earlier release than 3.24.0. If so, what version is it? Thank you!

**Comment by DarioS** [comment]
I hope this software will still be maintained. A lot of them aren't after they are published in a journal.

**Comment by AlexanderDilthey** [comment]
Yes, it will still be maintained! In fact I am currently in this process of rolling this out:
https://github.com/AlexanderDilthey/HLA-PRG-LA
